src/tasks/wifi.c

```diff
@@ -52,6 +55,8 @@
 #define STACK_SIZE		256
 /* Make all task names 16 chars including NULL char */
 #define THREAD_NAME		"WiFi Task      "
+/* The highest channel WiFi can use (USA) */
+#define WIFI_MAX_CHANNEL	11
 
```

Comment:
If we're doing channel clamping, we're going to need to build in some more sophisticated region management.  If we're selling into ETSI regions, we'll need to obey their rules.